src/DocumentationBrowserViewExtension/DocumentationBrowserViewExtension.cs

```diff
@@ -48,17 +48,18 @@ internal void OnMessageLogged(ILogMessage msg)
 
         #region IViewExtension lifecycle
 
-        public void Startup(ViewStartupParams viewLoadedParams)
+        public void Startup(ViewStartupParams p)
```

Comment:
Doesn't it make more sense to put a better name in the interface? Single letter params are not a good practice.

Reply:
@mmisol I agree, will update